interface/main/messages/messages.php

```diff
@@ -536,11 +536,11 @@ class='form-horizontal'
                                                 $body = preg_replace('/(:\d{2}\s\()' . $result['pid'] . '(\sto\s)/', '${1}' . $patientname . '${2}', $body);
                                                 $body = preg_replace('/(\d{4}-\d{2}-\d{2} \d{2}:\d{2}\s\([^)(]+\s)(to)(\s[^)(]+\))/', '${1}' . xl('to{{Destination}}') . '${3}', $body);
                                                 $body =nl2br(text(oeFormatPatientNote($body)));
-                                                echo "<div class='text oe-margin-t-3' style='border: 1px solid var(--gray); padding: 5px;'>" . $body . "</div>";
+                                                echo "<div class='text oe-margin-t-3 p-5' style='border: 1px solid var(--gray);'>" . $body . "</div>";
```

Comment:
`p-5` is too extreme here. I recommend running a docker environment and going from 1 to 5 to find the right padding for this.